src/tangy-eftouch/tangy-eftouch.html

```diff
@@ -0,0 +1,92 @@
+<link rel="import" href="/bower_components/polymer/polymer-element.html">
+
+<dom-module id="tangy-eftouch">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+    <div> [[statusmessage]] </div>
+    <slot></slot>
+  </template>
+
+  <script>
+    /**
+     * `tangy-timed`
+     * 
+     *
+     * @customElement
+     * @polymer
+     * @demo demo/index.html
+     */
+    class TangyEftouch extends Polymer.Element {
+      static get is() { return 'tangy-eftouch'; }
+      static get properties() {
+        return {
+          statusmessage: {
+            type: String,
+            value: 'Click start to begin'
+          },
+            display_sound_url: {
+            type: String,
+            value: 'hoot'
+          },
+            transition_sound_url: {
+            type: String,
+            value: 'hoot'
+          }
+        };
+      }
+
+      // Element class can define custom element reactions
+      // @TODO: Duplicating ready?
+      connectedCallback() {
+        super.connectedCallback();
+      }
+    
+      ready() {
```

Comment:
@chrisekelley I think using `ready` instead of `connectedCallback` may lead to some issues when the component touches the light dom children. Was looking into this a little yesterday.

> Don't use ready to initialize an element based on dynamic values, like property values or an element's light DOM children. Instead, use observers to react to property changes, and observeNodes or the slotchange event to react to children being added and removed from the element.

https://www.polymer-project.org/2.0/docs/devguide/custom-elements#one-time-initialization

I don't totally grok observers and I'm not sure why they don't mention connectedCallback as an alternative and I'm not sure what they mean by `The custom elements specification doesn't provide a one-time initialization callback. Polymer provides a ready callback, invoked the first time the element is added to the DOM.` because I'm under the impression that connectedCallback does that.

Anyways, figured I'd give you a heads up on this.